install/kubernetes/helm/istio/values.yaml

```diff
@@ -389,13 +389,10 @@ servicegraph:
   # prometheus addres
   prometheusAddr: http://prometheus:9090
 
-zipkin:
+tracing:
   enabled: false
   replicaCount: 1
-  image:
-    repository: docker.io/openzipkin/zipkin
-    tag: 2.7
-    pullPolicy: IfNotPresent
+  provider: zipkin
```

Comment:
I think we could default to jaeger, lower resources so easier to run in tests and demo. Real users can chose based on what they already use or want.